indico/web/client/js/react/components/pictures/PictureCropper.jsx

```diff
@@ -61,7 +61,7 @@ export function PictureCropper({cropperRef, src, onCrop, backAction, minCropSize
             styleName="cropper-action-btn"
             icon="check"
             primary
-            content={Translate.string('Done')}
+            content={Translate.string('Crop')}
```

Comment:
Just to add some more context for our translators
```suggestion
            // i18n: Crop an image
            content={Translate.string('Crop')}
```